.ci/bump-stack-version.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+#
+# Given the stack version this script will bump the version.
+#
+# This script is executed by the automation we are putting in place
+# and it requires the git add/commit commands.
+#
+set -euo pipefail
+MSG="parameter missing."
+VERSION=${1:?$MSG}
+
+OS=$(uname -s| tr '[:upper:]' '[:lower:]')
+
+if [ "${OS}" == "darwin" ] ; then
+	SED="sed -i .bck"
+else
+	SED="sed -i"
+fi
+
+echo "Update stack with version ${VERSION}"
+${SED} -E -e "s#(image: docker\.elastic\.co/.*):[0-9]+\.[0-9]+\.[0-9]+(-[a-f0-9]{8})?#\1:${VERSION}#g" docker-compose.yml
```

Comment:
this is going to cause https://github.com/elastic/apm-server/blob/master/script/check_docker_compose.sh to fail
I suppose we should just change that to check that the stack version is up to date, and disregard the build ID

Reply:
Addressed now -> https://github.com/elastic/apm-server/pull/5127/files#r618224858